source/ways-of-working/common-pipeline/common-pipeline.html.md.erb

```diff
@@ -89,6 +89,8 @@ End to end process:
     - updating the github flux repo with the new image name
 13. Flux will see new changes and deploy new pod
 
+In short, Jenkins runs "terraform plan" when you push, and "terraform apply" when you merge.
```

Comment:
not sure on this one, most runs are on the application pipeline which does a lot more than terraform plan and apply